examples/getting_started/example_save_and_load.py

```diff
@@ -111,11 +138,24 @@ def prepare_ocp(
     print(f"Final objective value : {np.nansum(sol.cost)} \n")
     sol.print()
 
-    # --- Save the optimal control program and the solution --- #
+    # --- Save the optimal control program and the solution with stand_alone = False --- #
     ocp.save(sol, "pendulum.bo")  # you don't have to specify the extension ".bo"
 
     # --- Load the optimal control program and the solution --- #
     ocp_load, sol_load = OptimalControlProgram.load("pendulum.bo")
 
     # --- Show results --- #
     sol_load.animate()
+    sol.graphs()
+
+    # --- Save the optimal control program and the solution with stand_alone = True --- #
+    ocp.save(sol, f"pendulum_sa.bo", stand_alone=True)
+
+    # --- Load the solution and add extra plots --- #
+    with open(f"pendulum_sa.bo", "rb") as file:
+        states, controls, parameters = pickle.load(file)
+    ocp.add_plot(
+        "My New Extra Plot",
+        lambda states, controls, parameters: custom_plot_callback(states, [0, 1, 2, 3]),
+        plot_type=PlotType.PLOT,
+    )
```

Comment:
After thinking about it, it does not make much sens to have an add_plot here since it will never be plotted anyway. And it won't work if put before the `sol_load.graphs()` either

Reply:
Done